server/container_create.go

```diff
@@ -589,6 +589,10 @@ func (s *Server) createSandboxContainer(ctx context.Context, containerID string,
 
 	containerImageConfig := containerInfo.Config
 
+	if containerImageConfig.Config.StopSignal != "" {
+		specgen.AddAnnotation("org.opencontainers.image.stopSignal", containerImageConfig.Config.StopSignal)
```

Comment:
in case you're wondering why I'm using this key: https://github.com/opencontainers/image-spec/pull/492/files#diff-8aafbe2c3690162540381b8cdb157112R57